TLM/TLM/UI/SubTools/LaneConnectorTool.cs

```diff
@@ -361,15 +374,15 @@ bool drawMarker
 
                         if (nodeMarkers != null) {
                             selectedMarker = null;
-
+                            int forwardSegmenIDX = GetFirstSegmentIDX(nodesBuffer[SelectedNodeId]);
```

Comment:
Naming: Typo in `segmen` and write `Index` unless its a some special abbreviation